client/lib/i18n-utils/utils.js

```diff
@@ -165,7 +165,7 @@ const urlLocalizationMapping = {
 };
 
 export function localizeUrl( fullUrl, locale ) {
-	const localeSlug = locale || getLocaleSlug();
+	const localeSlug = locale || ( typeof getLocaleSlug === 'function' ? getLocaleSlug() : 'en' );
```

Comment:
Not sure about this, under which circumstances is [`getLocaleSlug`](https://github.com/Automattic/i18n-calypso/blob/master/lib/index.js#L306) not a function?